terra_settings/views.py

```diff
@@ -18,3 +19,15 @@ def get(self, request):
         terra_settings.update(TERRA_APPLIANCE_SETTINGS)
 
         return Response(terra_settings)
+
+
+class BaseLayerView(APIView):
+    def get(self, request, *args, **kwargs):
+        baselayers = [
+            {
+                "id": baselayer.id,
+                "name": baselayer.name,
```

Comment:
why don't you add tilejson url here ?

Reply:
May be base layer list should be included in SettingsView, as settings view provide general settings for terra lego apps